opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/instrument.py

```diff
@@ -0,0 +1,156 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC
+from typing import Iterable, Union
+
+from opentelemetry import _metrics as metrics_api
+from opentelemetry.sdk._metrics._internal.instrument_descriptor import (
+    InstrumentDescriptor,
+)
+from opentelemetry.sdk._metrics._internal.measurement import (
```

Comment:
Should we remove the current [aggregation](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py and [instrument](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_metrics/instrument.py] files?